packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -804,29 +832,35 @@ void main() {
     await tester.tap(find.byType(DropdownMenu<TestMenu>));
     await tester.pump();
 
+    final bool isMobile = themeData.platform == TargetPlatform.android || themeData.platform == TargetPlatform.iOS || themeData.platform == TargetPlatform.fuchsia;
+    int expectedCount = isMobile ? 0 : 1;
+
     // Test onSelected on key press
     await simulateKeyDownEvent(LogicalKeyboardKey.arrowDown);
     await tester.pumpAndSettle();
     await tester.testTextInput.receiveAction(TextInputAction.done);
     await tester.pumpAndSettle();
-    expect(selectionCount, 1);
+    expect(selectionCount, expectedCount);
+    // The desktop platform close menu by entering 'done' button. So we need to reopen it.
```

Comment:
What 'done' button?

This should read something like: the desktop platform closed the menu when <something>.

Reply:
Sorry about the confusion. It should be the done action when editing is finished. Just updated this comment.